vignettes/epidist.Rmd

```diff
@@ -1,201 +1,250 @@
 ---
-title: "Getting Started with Epidist"
-description: "A quick start guide to using the Epidist package"
-author: Epidist Team
+title: "Getting started with epidist"
+description: "A quick start guide to using the epidist R package"
 output: 
-  bookdown::html_vignette2:
+  bookdown::html_document2:
     fig_caption: yes
     code_folding: show
+    number_sections: true
 pkgdown:
   as_is: true
-csl: https://raw.githubusercontent.com/citation-style-language/styles/master/apa-numeric-superscript-brackets.csl
+# csl: https://raw.githubusercontent.com/citation-style-language/styles/master/apa-numeric-superscript-brackets.csl
 link-citations: true
 vignette: >
-  %\VignetteIndexEntry{Getting Started with Epidist}
+  %\VignetteIndexEntry{Getting started with epidist}
   %\VignetteEngine{knitr::rmarkdown}
   %\VignetteEncoding{UTF-8}
+bibliography: references.bib
 ---
 
 ```{r setup, include=FALSE}
 # exclude compile warnings from cmdstanr
 knitr::opts_chunk$set(
-  fig.path = "figures/getting-started-nowcasting-",
-  cache = TRUE, dpi = 330,
-  collapse = TRUE, comment = "#>", out.width = "100%",
-  message = FALSE, warning = FALSE, error = FALSE,
-  eval = FALSE
+  fig.path = "figures/epidist-",
+  cache = TRUE,
+  collapse = TRUE,
+  comment = "#>",
+  message = FALSE,
+  warning = FALSE,
+  error = FALSE
 )
 ```
 
-# Quick start
-
-In this quick start, we demonstrate using `epidist` to ...
-
-# Package
-
-In this quick start, we also use `data.table`, `purrr`, and `ggplot2` packages. These are both installed as dependencies when `epidist` is installed. Note that all output from `epidist` is readily useable with  other tools, including `tidyverse` packages (see [here](https://mgimond.github.io/rug_2019_12/Index.html) for a comparison).
```

Comment:
Why cut?

Reply:
I don't think this is crucial information for someone reading the "get started" to be presented with at the beginning. If someone cares about dependencies then likely they know that they can look at the `DESCRIPTION` file to figure out the dependencies.

The point about output being readily usable with other tools seems somewhat obvious to me. Are people writing packages where the output is only usable with `data.table` say?

Reply:
> If someone cares about dependencies then likely they know that they can look at the DESCRIPTION file to figure out the dependencies.

Ah so this is because these aren't always going to be package dependencies. We might (and do here - looking at you `gt`) use packages that only appear in the suggests. This means that when people follow our install guides they won't always be able to run the vignette without either installing the suggests or manually knowing about them. 

> The point about output being readily usable with other tools seems somewhat obvious to me. Are people writing packages where the output is only usable with data.table say?

I think it might be obvious to a lot of people but I also think this could scare quite a large minority of people so at the cost of the single sentence seems worth keeping.

Reply:
Added:

Finally, to run this vignette yourself, you will need the `data.table`, `purrr` and `ggplot2` packages installed.
Note that to work with outputs from `epidist` you do not need to use `data.table`: any tool of your preference is suitable.